src/Tests/Microsoft.NET.TestFramework/Microsoft.NET.TestFramework.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
```

Comment:
Why is this showing up as a new file instead of as a diff against the existing csproj?

Reply:
likely because it was moved and modified enough that git no longer tracked it

Reply:
as long as your fine with the standalone commits not being buildable, I can do that.